web/html/src/build/fill-spec-file.js

```diff
@@ -10,7 +10,7 @@ function fillSpecFile() {
         }
         return item;
     });
-    const mappedProcessedLicenses = Array.from(new Set(processedLicenses)).sort().join(" and ");
+    const mappedProcessedLicenses = Array.from(new Set(processedLicenses)).sort().join(" AND ");
```

Comment:
Is this change also intended?

Reply:
Yes, this is for [SUSE/spacewalk#15002](https://github.com/SUSE/spacewalk/issues/15002), I either fix it here or mess up the licenses in master if this gets merged.

Reply:
ok, all good then :+1: